config/routes.rb

```diff
@@ -24,6 +24,8 @@
 
   resources :subscriptions, only: %i[new create edit update] do
     get :unsubscribe, on: :member
+    resources :feedback, only: %i[new update], controller: 'job_alert_feedback'
+    get 'feedback(/:id)/edit', to: 'job_alert_feedback#edit', as: 'edit_feedback'
```

Comment:
This looks to me like a standard edit action, why is it custom?

Reply:
Originally when I made it it was producing a pathname with 'index' instead of 'edit' and I couldn't work out why. But now it works as it's meant to.